src/boosting/gbdt.h

```diff
@@ -180,6 +199,10 @@ class GBDT: public Boosting {
   data_size_t bag_data_cnt_;
   /*! \brief Number of traning data */
   data_size_t num_data_;
+  /*! \brief Number of classes */
+  int num_class_;
+  /*! \brief Current class */
+  int curr_class_;
```

Comment:
I know this. My point is should we really need to use "curr_class_" as a member variable?
why not pass "curr_class" as the argument of these functions?